src/libstore/derivations.hh

```diff
@@ -87,10 +88,42 @@ Derivation readDerivation(const Store & store, const Path & drvPath);
 // FIXME: remove
 bool isDerivation(const string & fileName);
 
-Hash hashDerivationModulo(Store & store, const Derivation & drv, bool maskOutputs);
+/* Returns hashes with the details of fixed-output subderivations
+   expunged.
+
+   A fixed-output derivation is a derivation whose outputs have an a
+   specified content hash hash algorithm. (Currently they must have
```

Comment:
```suggestion
   A fixed-output derivation is a derivation whose outputs have a
   specific content hash hash algorithm. (Currently they must have
```

(or `the specified`)

Would this still make sense if one  of the `hash`es was dropped -- `specific content hash algorithm`?

Reply:
It was supposed to be "hash and hash algorithm", roughly like the old version of the comment.